### ISSUE TITLE
Ingest Variants determined by Sanger Sequencing

### ISSUE THREAD

**Issue by your-highness** [bihealth/varfish-server#578]
**Is your feature request related to a problem? Please describe.**
Many clinical relevant regions are difficult to sequence via NGS due to low complexity and homology (see #529). Usually, these diagnostics gaps are filled by PCR amplification and subsequent Sanger sequencing. This information is currently not uploaded to VarFish.

**Describe the solution you'd like**
One could harness https://github.com/gear-genomics/indigo/blob/main/indigo.sh for automated SNV/InDel detection from raw sequencing traces (_e.g._ `.abi` files). Results can be saved as BCF/VCF. Most importantly, the import of these variants should be decoupled from NGS variant calls to assess variants correctly in downstream variant analysis.

**Describe alternatives you've considered**
An alternative is to append Indigo calls to the NGS VCF. However, this approach does not trace the origin of the variant calls.

**Additional context**
Honestly, the variant calling from Sanger sequencing requires alot of expert intervention and - despite the existence of reliable tools - most experiments have to be studied by experts visually as well. A linkout to an Sanger analysis program with the raw trace loaded would solve this problem.

**Comment by holtgrewe** [comment]
I had a quick look at [indigo](https://www.gear-genomics.com/indigo/) and clicked on "Example". Is the output of this suitable for interpretation by expert users already?

**Comment by your-highness** [comment]
> I had a quick look at [indigo](https://www.gear-genomics.com/indigo/) and clicked on "Example". Is the output of this suitable for interpretation by expert users already?

Yes. A linkout to a Indigo server would be great to visualize Sanger sequencing results for expert assessment.

To seperate NGS from Sanger results one could seperately import Sanger results as TSV that will be marked by an INFO field that seperate NGS from Sanger calls.

BTW: Indigo is the web-server based on [Tracy](https://link.springer.com/article/10.1186/s12864-020-6635-8)
